WordPress/Tests/Theme/NoFaviconUnitTest.inc

```diff
@@ -0,0 +1,6 @@
+/* ----- Bad ------- ERROR ---- */
+<link rel="shortcut icon" href="<?php myfunction('link' ); ?>">
```

Comment:
Please make the test files lead by example and not include other bad practices, the href functions should all have `esc_url()`s around them. Alternatively just hardcode a fake url.